src/common.css

```diff
@@ -18,6 +18,9 @@
 }
 
 /* Select2 theme */
+.select2-input {
+    height: 27px !important;
```

Comment:
Because the select2's css use "height: auto !import", it will be overwrited.
But now my css rule doesn't work, it will be overwrited too.